### DISCUSSION TITLE
How to restart a spark cluster after idle timeout?

### DISCUSSION THREAD

**Discussion by HaoYang670** [apache/kyuubi#5046] [Q&A] [answered]
Is there a way in Kyuubi to restart the Spark cluster using same Spark configs after idle timeout?

**Comment by HaoYang670** [comment]
@pan3793 Could you please take a look if you have time?

**Reply by pan3793** [reply, marked as answer]
There is no such functionality yet.

**Reply by HaoYang670** [reply]
Ok thank you for the clarification
